src/main/java/com/ericsson/eiffel/remrem/publish/cli/CliOptions.java

```diff
@@ -125,6 +133,20 @@ public static void handleMessageBusOptions(){
     		String key = PropertiesConfig.MESSAGE_BUS_PORT;
     		System.setProperty(key, exchangeName);
     	}
+  
+    	if (commandLine.hasOption("tls")) {
+    		String tls_ver = commandLine.getOptionValue("tls");
+    		if (tls_ver == null) {
+    			tls_ver = "NULL";
+    		}
+    		String[] validTlsVersions = new String[]{"1", "1.1", "1.2", "default"};
+    		if (StringUtils.indexOfAny(tls_ver, validTlsVersions) == -1) {
+    			throw new Exception("Specified TLS version is not valid! Specify a valid TLS version!");
```

Comment:
Not a good practice to throw the base class Exception. Use ParseException from apache commons cli. Also change where the handleMessageBusOptions are called to catch ParseException.